WeatherPlannerDemo/Features/CityList/Data/Repository/CityListRepository.swift

```diff
@@ -17,18 +18,13 @@ class CityListRepository: CityListRepositoryProtocol {
         self.reachability?.startReachability()
     }
     
-    func fetchCities(
-        _ coordinate: City.Coordination,
-        range: Int,
-        resultHandler: @escaping ([City]) -> Void
-    ) {
+    func fetchCities(_ coordinate: City.Coordination, range: Int) -> Observable<CitiesInCircle> {
         switch reachability?.connection {
         case .unavailable:
-            fetchCities(resultHandler: resultHandler)
+            //fetchCities(resultHandler: resultHandler)
+            return Observable<CitiesInCircle>.never()
```

Comment:
This will be refactored with PR for CoreData.

Reply:
yes!!!!!!

Reply:
:O